string cmd
int64 consumerReceivedTime
string destination
int64 producerSentTime
string source
